papermill_origami/noteable_dagstermill/ops.py

```diff
@@ -465,12 +363,12 @@ def define_noteable_dagstermill_op(
             "kind" not in tags,
             "user-defined solid tags contains the `kind` key, but the `kind` key is reserved for use by Dagster",
         )
-    default_tags = {"notebook_path": notebook_path, "kind": "ipynb"}
+    default_tags = {"notebook_path": notebook_url, "kind": "noteable"}
```

Comment:
switching this to the notebook url will make the View Notebook button open the notebook in a new tab. Changing kind to "noteable" will display the noteable logo in dagit instead of the `ipynb` tag